specs/OracleQueries.yml

```diff
@@ -307,22 +373,17 @@ components:
         - town
         - county
         - postcode
-        - case_reference
-        - case_type
-        - bankruptcy_type
-        - start_date
-        - debtor_discharge_date
-        - trustee_discharge_date
       properties:
+        ephemeral_officer_key:
+          type: string
+          description: The ephemeral UUID of the officer.
+          example: "B67053D91CC750DEE05400144FFBDD12"
```

Comment:
If is a UUID  with hashes
could use if desired

      type: string    
      description: blah blah
      format: uuid
      example: "b72596ce-da4e-41f9-bda5-2e52eeff8730"

Reply:
Updated to reference guid instead of uuid

Reply:
Changed to reference only string per conversation with @gn404

Reply:
Worth updating the readme with new endpoint info